sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -55,40 +56,40 @@ and then include the direct dependency in the dependencies section without the v
 If you want to take dependency on a particular version of the library that is not present in the BOM,
 add the direct dependency to your project as follows.
 
-[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;dependency})
+[//]: # ({x-version-update-start;com.azure:azure-ai-textanalytics;current})
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-ai-textanalytics</artifactId>
-    <version>5.1.9</version>
+    <version>5.2.0-beta.3</version>
```

Comment:
curios, this goes form a GA to a beta. does Java usually show the latest? or the latest stable library

Reply:
Usually java shows the latest stable version in the main and release the beta in feature branch. The feature branch will merge to main branch at GA. But since TA is merging the latest beta changes to main branch. I will update the version to beta for the beta features, otherwise the custom features should not include in the README.